src/MyEventEmitter.js

```diff
@@ -1,14 +1,70 @@
 'use strict';
 
 class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+  constructor() {
+    this.listeners = {};
+  }
+
+  on(event, callback) {
+    if (!this.listeners[event]) {
+      this.listeners[event] = [];
+    }
+
+    this.listeners[event].push(callback);
+  }
+
+  once(event, callback) {
+    const wrapper = (...args) => {
+      callback(args);
+      this.off(event, wrapper);
+    };
+
+    this.on(event, wrapper);
+  }
+
+  off(event, callback) {
+    if (!this.listeners[event]) {
+      return;
+    }
+
+    this.listeners[event] = this.listeners[event].filter((v) => v !== callback);
+  }
+
+  emit(event, ...args) {
+    if (!this.listeners[event]) {
+      return;
+    }
+
+    for (const listener of this.listeners[event]) {
+      listener(...args);
+    }
+  }
+
+  prependListener(event, callback) {
+    if (!this.listeners[event]) {
+      this.listeners[event] = [];
+    } else {
+      this.listeners[event].unshift(callback);
+    }
+  }
+  prependOnceListener(event, callback) {
+    const wrapper = (...args) => {
+      callback(args);
```

Comment:
Similar to line 18, the `callback` function is called with an array `args`. Use `callback(...args)` to ensure the callback receives individual arguments.